torchgeo/datasets/levircd.py

```diff
@@ -225,20 +225,34 @@ def plot(
 
         .. versionadded:: 0.2
         """
-        image1, image2, mask = (sample["image1"], sample["image2"], sample["mask"])
         ncols = 3
 
+        def get_rgb(img: Tensor) -> "np.typing.NDArray[np.uint8]":
+            img = img.permute(1, 2, 0).float().numpy()
```

Comment:
Before this line, `img` is a Tensor. After this line, `img` is an NDArray. You should use a different variable name instead of dynamic typing.